R/ecdc.R

```diff
@@ -20,37 +20,29 @@
 #' get_ecdc_cases(countries = "France")
 #'
 #' ## Code
-get_ecdc_cases <- function (countries = NULL){
```

Comment:
We probably don't need to @export this if using inside a wrapper.

Reply:
fixed